readyplayerme/rpm-unity-sdk-core
# [SDK-700] Add asset type filter

## [SDK-700](https://ready-player-me.atlassian.net/browse/SDK-700)

## Description

- Added filter for asset type



## How to Test

-  Check the SimpleAvatarCreator assetSelectionElements and observe the asset type enum showing correct types



## Checklist








[SDK-700]: https://ready-player-me.atlassian.net/browse/SDK-700?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
